src/__tests__/user.spec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable node/no-unpublished-require */
+const mongoose = require('mongoose');
+const supertest = require('supertest');
+const app = require('../app');
+const UserModel = require('../models/user');
+const connectDatabase = require('../db/connection');
+
+beforeAll(async () => {
+  connectDatabase();
+});
+
+afterAll(async () => {
+  await UserModel.deleteMany({});
+  await mongoose.disconnect();
+  await mongoose.connection.close();
+});
+
+describe('GET /api/user/:id', () => {
```

Comment:
You might want to write test where it also returns a user because the test you wrote is for returning an error

Reply:
The test does return a user, a deleted one to be precise, but the error is about token verification.

Reply:
why would you return a deleted user specifially?

Reply:
To be honest, it does not matter at all. Because the output would always be a 401, not authorized, I used the only id I had copied to my local. If it did return `User not found`, I would add some different cases: an available user id.